pkg/provider/github/github_test.go

```diff
@@ -119,6 +119,30 @@ func TestGithubSplitURL(t *testing.T) {
 			wantRef:  "main",
 			wantPath: "testdatas/remote_task.yaml",
 		},
+		{
+			name:     "Split URL with slash in branch",
+			url:      "https://github.com/openshift-pipelines/pipelines-as-code/blob/feature%2Fbranch/testdatas/remote_task.yaml",
```

Comment:
does it need to be `%20F`? or this is correct too?
```suggestion
			url:      "https://github.com/openshift-pipelines/pipelines-as-code/blob/feature%2F0branch/testdatas/remote_task.yaml",
```

Reply:
it's a mistake in doc it's %2f :) thanks will amend